draft-ietf-quic-recovery.md

```diff
@@ -890,8 +891,9 @@ follows:
    min_rtt = infinite
    time_of_last_sent_ack_eliciting_packet = 0
    time_of_last_sent_crypto_packet = 0
-   largest_sent_packet = 0
-   largest_acked_packet = 0
+   for pn_space in packet number spaces:
```

Comment:
I think we should define packet number spaces more clearly above, and possibly call it kPacketNumberSpaces.  I think it makes sense to define it as a list of 3 enum values.

Reply:
Not sure if we need that constant, since we're only looping over the spaces once (and only for initialization), but I introduced the enum.